agent/engine/dependencygraph/graph.go

```diff
@@ -73,7 +73,8 @@ func dependenciesCanBeResolved(target *api.Container, by []*api.Container) bool
 	}
 
 	return verifyStatusResolvable(target, nameMap, neededVolumeContainers, volumeCanResolve) &&
-		verifyStatusResolvable(target, nameMap, linksToContainerNames(target.Links), linkCanResolve)
+		verifyStatusResolvable(target, nameMap, linksToContainerNames(target.Links), linkCanResolve) &&
+		verifyStatusResolvable(target, nameMap, target.SteadyStateDependencies, onSteadyStateCanResolve)
```

Comment:
Can this line be removed, as `SteadyStateDependencies` isn't updated anywhere? Also do we need to check the `TransitionDependencySet` here?

Reply:
No, `SteadyStateDependencies` are still set via old state files if we read them in.

We probably should check `TransitionDependencySet` here too, but I'd rather not block on that right now.